src/features/users/user-controller.ts

```diff
@@ -154,7 +142,7 @@ class UserController {
             await emailHelper.sendEmail({
                 content: `Hello,
 
-                Please verify your account by clicking this url: ${baseUrl}/users/verify?verify_token=${newUser.verify_token}
+                Please verify your account by clicking this url: ${baseUrl}/users/verify?verify_token=${newUser.verifyToken}
                 `,
                 email: newUser.email,
                 subject: 'Please veryify account'
```

Comment:
Spelling error (should be fixed in this PR).
```suggestion
                subject: 'Please verify your Rederly account'
```